backbone.queryparams.js

```diff
@@ -309,8 +312,14 @@ function iterateQueryString(queryString, callback) {
   var keyValues = queryString.split('&');
   _.each(keyValues, function(keyValue) {
     var arr = keyValue.split('=');
-    callback(arr.shift(), arr.join('='));
+    if (!containsInvalidKey(keyValue)) {
```

Comment:
Do we need to check both key and value? Or would it be good enough to simply check the key for forbidden names? 
This could even have negative side-effects, if we flat out forbid a value to be e.g. "constructor", no?

Reply:
That's right, query parameter keys only are affected. This should only apply to keys. I have updated the code and test to reflect that. Thank you!